go.mod

```diff
@@ -1,11 +1,15 @@
 module github.com/dell/gobrick
 
-go 1.13
+go 1.17
```

Comment:
Is this version upgrade not going to break anything, because this is a big jump from 1.13 to 1.17

Reply:
works fine on testing. Will be running more regression tests